services/telegram/service.go

```diff
@@ -33,6 +34,11 @@ func NewService(c Config, d Diagnostic) *Service {
 		diag: d,
 	}
 	s.configValue.Store(c)
+	s.clientValue.Store(&http.Client{
```

Comment:
any reason why you decided to use `atomic.Value` instead of just using a mutex and a `*http.Client`?

Reply:
👍